components/WrapperCanvas.vue

```diff
@@ -1,16 +1,25 @@
 <script setup>
+    import Comic from '~/utils/Classes/Comic';
+    import ElementDS from '~/utils/Classes/Element';
+    import Text from '~/utils/Classes/Text';
+    import Asset from '~/utils/Classes/Asset';
+
     const props = defineProps({
         height: Number,
         panel: Object,
-        selectedId: String,
+        panelIsActive: Boolean,
+        comic: Object,
         lockAspectRatio: Boolean,
+        panelIndex: Number,
+        arrayBB: Object,
```

Comment:
I am quite confused on what this is. I would ask you to rename it to something ore telling.